src/appleseed/renderer/kernel/rendering/renderercomponents.cpp

```diff
@@ -364,6 +365,10 @@ bool RendererComponents::create_pixel_renderer_factory()
 
         return true;
     }
+    else if (name == "adaptive_tile")
```

Comment:
That doesn't make sense: `pixel_renderer` will simply not be set when using the adaptive tile renderer. You must remove this branch.

Reply:
I forgot sry.